requirements.txt

```
@@ -2,7 +2,7 @@ aiofiles==23.2.1
 aiohttp==3.8.5
 discord.py==2.3.2
 packaging==23.1
-pydantic==2.3.0
+pydantic==2.4.0
```

```suggestion
pydantic==2.4.1
```
[pydantic 2.4.1](https://github.com/pydantic/pydantic/releases/tag/v2.4.1) has been released, if an update is to be done it should probably be to that.

---

So is 2.4.2, maybe we should upgrade directly to that instead?

---

didn't even notice it was out as it isn't on their [changelog page](https://docs.pydantic.dev/latest/changelog/), but yeah that'd be better.